source/compiler/qsc_qasm/src/tests/statement/gate_call.rs

```diff
@@ -232,6 +231,7 @@ fn barrier_can_be_called_without_qubits() -> miette::Result<(), Vec<Report>> {
 fn barrier_generates_qir() -> miette::Result<(), Vec<Report>> {
     let source = r#"
         include "stdgates.inc";
+        #pragma qdk.qir.profile Adaptive_RI
         bit[1] c;
         qubit[2] q;
         barrier q[0], q[1];
```

Comment:
Does the profile support any casing or does it have to be exactly cased like all of these samples? The error message shows the values lower case.